.github/workflows/latestPublish.yml

```diff
@@ -4,7 +4,7 @@ on:
       - '*'
   # Scheduled jobs will only run on the default branch
   schedule:
-    - cron: '* */5 * * *' # Run every 5 hours
+    - cron: '0 */5 * * *' # Run every 5 hours
```

Comment:
I think this needs to be a zero. Right now it's running every 9 minutes for some reason